ipaserver/plugins/cert.py

```diff
@@ -961,14 +961,12 @@ def _emails_are_valid(csr_emails, principal_emails):
     appear in ldap entry, comparing the domain part case-insensitively.
     """
 
-    if not any(principal_emails) or not any(csr_emails):
-        return False
-
     def lower_domain(email):
-        email_splited = email.split('@', 1)
-        email_splited[1] = email_splited[1].lower()
+        email_splitted = email.split('@', 1)
+        if len(email_splitted) > 1:
```

Comment:
This became necessary since we don't have the if statement (in line 964) anymore.